Gemfile

```diff
@@ -14,6 +14,7 @@ gem "log_formatter", "~> 0.8.2"
 gem "puma", "~> 4.3"
 gem "sentry-raven", "~> 3.0"
 gem "uglifier", "~> 4.2"
+gem "sassc-rails"
```

Comment:
think this'll need to be pinned for it to work

Reply:
argh, of course

Reply:
Now changed to `sass-rails` and pinned to less than 6.